autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/ui/credential/management/AutofillCharacterValidator.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.autofill.ui.credential.management
+
+import com.duckduckgo.di.scopes.AppScope
+import com.squareup.anvil.annotations.ContributesBinding
+import javax.inject.Inject
+
+interface AutofillCharacterValidator {
+    fun isLetter(character: Char): Boolean
+}
+
+@ContributesBinding(AppScope::class)
+class LatinCharacterValidator @Inject constructor() : AutofillCharacterValidator {
+
+    override fun isLetter(character: Char): Boolean {
+        return character in 'a'..'z' || character in 'A'..'Z'
```

Comment:
Question: Why are we ignoring the non-latin characters?

Reply:
Asked for clarification around grouping/sorting rules in [UI Question: Sorting and Grouping of non-latin characters in titles or domains](https://app.asana.com/0/1202688973961658/1202688973961658/f)

Will go ahead with the code as is for now in this PR and address later depending on the answer to the above